pkg/rtc/participant.go

```diff
@@ -975,11 +974,13 @@ func (p *ParticipantImpl) onMediaTrack(track *webrtc.TrackRemote, rtpReceiver *w
 
 	publishedTrack, isNewTrack := p.mediaTrackReceived(track, rtpReceiver)
 
-	p.params.Logger.Infow("mediaTrack published",
-		"kind", track.Kind().String(),
-		"trackID", publishedTrack.ID(),
-		"rid", track.RID(),
-		"SSRC", track.SSRC())
+	if publishedTrack != nil {
+		p.params.Logger.Infow("mediaTrack published",
+			"kind", track.Kind().String(),
+			"trackID", publishedTrack.ID(),
+			"rid", track.RID(),
+			"SSRC", track.SSRC())
+	}
```

Comment:
This is the crash @lukasIO reported with publish failing due to no pending track